protos/core.proto

```diff
@@ -52,6 +52,16 @@ message Target {
   bool include_dependent_assertions = 4;
 }
 
+enum FileFormat {
+  UNKNOWN = 0;
```

Comment:
Let's follow the same pattern on all enums. Should be `FILE_FORMAT_UNSPECIFIED`.